### DISCUSSION TITLE
Why can I connect to a Moonraker on port 80?

### DISCUSSION THREAD

**Discussion by AndreKR** [fluidd-core/fluidd#1185] [Q&A] [answered]
I have two Moonraker instances on my Klipper machine, one on `http://192.168.0.36:7125` and one on `http://192.168.0.36:7125`. I have both set up in fluidd with the mentioned addresses and I can switch between them. I can see in the dev tools that fluidd opens a websocket connection to `ws://192.168.0.36:7125` or `ws://192.168.0.36:7126` respectively. All great.

![image](https://github.com/fluidd-core/fluidd/assets/1188538/3b03a1ae-f599-453b-9e72-a18f3499a6f8)

What confuses me: I have a third entry in my fluidd hosts: `http://192.168.0.36`. When I select it, I can see how fluidd opens a websocket connection to `ws://192.168.0.36` (port 80) and somehow a Moonraker answers! (I think it's the one that actually runs on 7125.)

What gives?

**Comment by pedrolamas** [comment, marked as answer]
That is because port 80 is being served by Nginx, so it can redirect specific paths to other places (in this case, to other Moonraker instances)!

For example, Moonraker use paths like /server, so that path is mapped to a Moonraker instance on 7125.
